rest_docs/rest-catalog-open-api.yaml

```diff
@@ -1381,15 +1381,35 @@ components:
               } }
 
     CreateNamespaceResponse:
-      description: A succesful call to create a namespace
+      description:
+        Represents a successful call to create a namespace.
+        Returns the namespace created, as well as any properties that were stored for the namespace,
+        including those the server might have added. Implemenations are not required to support namespace
+        properties.
       content:
         application/json:
           schema:
             type: object
+            required:
+              - namespace
             properties:
-              created:
-                type: boolean
-                description: true if the namespace was added to the catalog
+              namespace:
+                $ref: '#/components/schemas/Namespace'
+              properties:
+                type: object
+                additionalProperties:
+                  type: string
+                description:
+                  Properties stored on the namespace, if supported by the server.
+                  If the server does not support namespace properties, it should return null for this field.
```

Comment:
What is the benefit of requiring `null`? Should this be explicitly null, like `"properties": null` or can it be omitted?

Reply:
Yeah I'm actually rethinking that. This seems like an arbitrary requirement.

Originally my thinking was to see if the server supported it or not, but that can really be an implementation detail and left up to them.

In the PR for adding serializers,  explicit `null` and omitting the property are both fine and parse correctly. The final object winds up with an empty map if calling the getter method on it.